packages/vue/package.json

```diff
@@ -38,6 +38,10 @@
   "publishConfig": {
     "access": "public"
   },
+  "files": [
+    "/dist",
+    "/src"
```

Comment:
So, with some changes we made, dist and src aren't automatically part of the build ... you can see this by checking `build.sh` and the ouput by running `npm run build` (we deploy the dist directory) ... if we want to include src (maybe not required?) we'll need to copy it into dist as part of `build.sh`